// @imagina/qsite/_i18n/master/en-us/ui.js
export default {
  label: {
    delete: 'Delete',
    cancel: 'Cancel',
    open : 'Open',
    close: 'Close',
    closed : 'Closed',
    set: 'Set',
    ok: 'Ok',
    select: 'Select',
    reset: 'Reset',
    remove: 'Remove',
    update: 'Update',
    create: 'Create',
    search: 'Search',
    filter: 'Filter | Filters',
    refresh: 'Refresh',
    move: 'Move',
    clear: 'Clear',
    loading: 'Loading',
    validating: 'Validating',
    save: 'Save',
    confirm: 'Confirm',
    configuration: 'Configuration | Configurations',
    role: 'Role | Roles',
    department: 'Department | Departments',
    language: 'Language | Languages',
    interface: 'Interface | Interfaces',
    currency: 'Currency | Currencies',
    setting: 'Setting | Settings',
    tag: 'Tag | Tags',
    description: 'Description | Descriptions',
    option: 'Option | Options',
    edit: 'Edit',
    addFile: 'Add File',
    error: 'Error',
    enabled: 'Enable',
    disabled: 'Disable',
    available : 'Available',
    soldOut: 'Sold Out',
    all : 'All',
    data : 'Data',
    link : 'Link',
    image: 'Image | Images',
    add : 'Add',
    required : 'Required',
    yes : 'Yes',
    no : 'No',
    permission : 'Permission | Permissions',
    allow : 'Allow',
    deny : 'Deny',
    inherit : 'Inherit',
    day : 'Day | Days',
    monday : 'Monday',
    tuesday : 'Tuesday',
    wednesday : 'Wednesday',
    thursday : 'Thursday',
    friday : 'Friday',
    saturday : 'Saturday',
    sunday : 'Sunday',
    hour : 'Hour | Hours',
    minute : 'Minute | Minutes',
    second : 'Second | Seconds',
    home : 'Home',
    find : 'Find',
    user : 'User',
    impersonate : 'Impersonate',
    address : 'Address | Addresses',
    contacts : 'Contact | Contacts',
    socialNetwork : 'Social Network | Social Networks',
    success : 'Success',
    ready : 'Ready',
    module: 'Module',
    entity : 'Entity',
    change : 'Change',
    socialNetwork : 'Social Network | Social Networks',
    logo : 'Logo | Logos',
    pending: 'Pending | Pendings',
    notification : 'Notification | Notifications',
    leave : 'Leave',
    action : 'Action | Actions',
    view : 'View | Views',
    grid : 'Grid | Grids',
    table : 'Table',
    expired : 'Expired',
    empty : 'Empty',
    completed : 'Completed | Completed'
  },
  message: {
    notFound: 'Sorry, nothing here',
    goToHome: 'Go to home',
    deleteRecord: 'Are you sure, you want to delete this record?',
    formInvalid: 'There are missing fields to complete, check the form',
    fieldRequired: 'This field is required',
    fieldEmail : 'Must have email format',
    fieldCheckPassword : 'Does not match the password',
    fieldMinLeng : 'Must contain at least {num} digits',
    updatePassword : 'Update password',
    recordCreated: 'Record created',
    recordUpdated: 'Record updated',
    recordDeleted: 'Record deleted',
    recordNoCreated: 'Record NOT created',
    recordNoUpdated: 'Record NOT updated',
    recordNoDeleted: 'Record NOT deleted',
    errorRequest : 'Error requesting data',
    saveAndReturn : 'Save and Return',
    saveAndEdit : 'Save and continue editing',
    saveAndCreate : 'Save and create new',
    searchByIdOrName : 'Search by ID or Name',
    searchNotFound : 'Search not found',
    uploadImage : 'Upload image',
    changeImage : 'Change image',
    editImage : 'Edit image',
    deleteImage : 'Delete image',
    captcha : 'This site is protected by reCAPTCHA and the Google '+
      '<a href="https://policies.google.com/privacy" target="_blank">Privacy Policy</a> and ' +
      '<a href="https://policies.google.com/terms" target="_blank">Terms of Service</a> apply.',
    requiredCaptcha : 'Captcha is required',
    deniedPermissions: "You don't have the necessary permissions"
  },
  form: {
    id : 'ID',
    title: 'Title | Title',
    caption: 'Caption | Captions',
    firstName : 'First Name | First Name',
    lastName : 'Last Name | Last Name',
    password : 'Pasword | Paswords',
    checkPassword : 'Check Pasword | Check Paswords',
    name: 'Name | Names',
    email: 'Email | Emails',
    slug: 'Slug',
    summary: 'Summary | Summaries',
    description: 'Description Descriptions',
    category: 'Category | Categories',
    parent: 'Parent | Parents',
    status: 'status | Statuses',
    image: 'Image | Images',
    video: 'Video | Videos',
    firstImage: 'First Image',
    secondaryImage: 'Secondary Image',
    gallery: 'Gallery | Galleries',
    tag: 'Tag | Tag',
    publish: 'Published',
    noPublish: 'Not Published',
    draft: 'draft',
    pending: 'Pending',
    metaTitle: 'Meta Title',
    metaDescription: 'Meta Description',
    sku : 'Sku',
    stock: 'stock',
    price : 'Price | Prices',
    quantity : 'Quantity | Quanitities',
    weight : 'Weight',
    length : 'Length',
    height : 'Height',
    width : 'Width',
    point : 'Point | Points',
    reference : 'Reference',
    option: 'Option | Options',
    type : 'Type | Types',
    required : 'Required',
    createdAt : 'Created At',
    updatedAt : 'Updated At',
    actions : 'Actions',
    sort : 'Sort',
    prefix : 'Prefix',
    currentNumber : 'Current number',
    firstNumber : 'First number',
    lastNumber : 'Last number',
    resolution : 'Resolution',
    autoincrement : 'autoincrement',
    default : 'Default',
    percentage : 'Porcentage',
    address : 'Address | Addresses',
    phone : 'Phone | Phones',
    schedule : 'Schedule | Schedules',
    menu : 'Menu | Menus',
    author : 'Author | Authors',
    icon : 'Icon | Icons',
    masterRecord : 'Master record',
    birthday : 'Birthday',
    identification : 'Identification',
    city : 'City | cities',
    state : 'State | State',
    zipCode : 'Zip Code',
    socialNetwork : 'Social Network | Social Networks',
    startDate : 'Start Date',
    endDate : 'End Date',
    code : 'Code | Codes',
    increase : 'Increase',
    date : 'Dates | Dates',
    dueDate : 'Due date',
    tax : 'Tax | Taxes',
    orderBy : 'Order by',
    orderWay : 'Order way',
    assignedTo : 'Assigned to',
    comment : 'Comentario | Comentarios'
  },
  configList: {
    loggedAs: 'logged as',
    fullScreen: 'show in full screen',
    signOut: 'sign out',
    leaveImpersonating : 'Leave impersonating'
  },
}
