// @imagina/qcommerce/_i18n/en-us/layout.js
export default {
  newProduct : 'New Product',
  updateProduct : 'Update Product',
  newCategory : 'New Category',
  updateCategory : 'Update Category',
  productOptions : 'Product Options',
  newOption : 'New Option',
  updateOption : 'Update Option',
  newOptionValue : 'New Option value',
  updateOptionValue : 'Update Option Value',
  newProductOptionValue : 'New Product Option Value',
  updateProductOptionValue : 'Update Product Option Value',
  updatePaymentMethod : 'Update Payment Method',
  updateShippingMethod : 'Update Shipping Method',
  newOrder : 'New Order',
  updateOrder : 'Update Order',
  newCoupon: 'New Coupon',
  updateCoupon: 'Update Coupon',

  orderAccountInformation: 'Order & Account Information',
  orderStatus: 'Order Status',
  orderDate: 'Order Date',
  placedFromIP: 'Placed from IP',
  accountInformation: 'Account Information',
  addressInformation: 'Address Information',
  billingAddress: 'Billing Address',
  paymentShippingMethod: 'Payment & Shipping Method',
  paymentInformation: 'Payment Information',
  itemsOrdered: 'Items Ordered',
  orderTotal: 'Order Total',
  subtotal: 'Subtotal',
  shipping: 'Shipping',
  orderHistory: 'Order History',

  form : {
    availableDate : 'Available From',
    minimumOrder : 'Minimum Order',
    orderWeight : 'Weight Order',
    requriedShipping : 'Shipping Required',
    freeShipping : 'Free Shipping',
    subtractFromStock : 'Subtract from Stock',
    relatedProducts : 'Related Products',
    optionValue : 'Option Value',
    parentOptionValue : 'Parent Option Value',
    productOptionValues : 'Product Option Values',
    showInMenu:'Show in menu',
    apiKey : 'API Key',
    apiLogin : 'API Login',
    transactionKey : 'Transaction Key',
    clientKey : 'Client Key',
    clientSecret : 'Client Secret',
    merchantId : 'Merchant ID',
    mode : 'Mode',
    accountId : 'Account ID',
    allowTestCreditCard : 'Test Credit Card',
    cost : 'Cost',
    minimum : 'Minimum',
    deliveryFee : 'Delivery Fee',
    userId : 'User ID',
    zipCode : 'Zip Code',
    shippingRates : 'Shipping Rates',
    machinable : 'Machinable',
    accessKey : 'Access Key',
    shipperPostalCode : 'Shipper Postal Code',
    shipperStateCode : 'Shipper State Code',
    shipperCountryCode : 'Shipper Country Code',
    total: 'Total',
    city: 'City',
    country: 'Country',
    shippingMethod: 'Shipping Method',

    // Coupons
    code: 'Coupon Code',
    type: 'Type',
    category: 'Category',
    product: 'Product',
    customer: 'Customer',
    discount: 'Discount',
    typeDiscount: 'Type Discount',
    logged: 'Logged Required',
    shipping: 'Shipping',
    dateStart: 'Date Start',
    dateEnd: 'Date End',
    quantityTotal: 'Quantity Total',
    quantityTotalCustomer: 'Quantity Total Per Customer',
    status: 'Status',

  },
  message : {
    warnAddOpt : 'To add options, you must first create the product',
    noOptions : 'There are no added options',
    noDeleteOption : 'You can not delete this option, you are the father of {name}',
    selectProduct : 'Select a product option'
  }
}
