// @imagina/qcommerce/_config/backendPages.js
export default {
  //Product List
	products: {
		permission: 'icommerce.products.index',
		activated: true,
		path: '/ecommerce/products',
		name: 'qcommerce.admin.products.index',
		page: () => import('@imagina/qcommerce/_pages/admin/products/index'),
		layout: () => import('@imagina/qsite/_layouts/master.vue'),
		title: 'qcommerce.sidebar.adminProducts',
		icon: 'fas fa-boxes',
		authenticated: true,
	},
	//Product Create
	productCreate: {
		permission: 'icommerce.products.create',
		activated: true,
		path: '/ecommerce/products/create',
		name: 'qcommerce.admin.products.create',
		page: () => import('@imagina/qcommerce/_pages/admin/products/form'),
		layout: () => import('@imagina/qsite/_layouts/master.vue'),
		title: 'qcommerce.sidebar.adminProductCreate',
		icon: 'fas fa-boxes',
		authenticated: true,
	},
	//Product Update
	productEdit: {
		permission: 'icommerce.products.edit',
		activated: true,
		path: '/ecommerce/products/:id',
		name: 'qcommerce.admin.products.edit',
		page: () => import('@imagina/qcommerce/_pages/admin/products/form'),
		layout: () => import('@imagina/qsite/_layouts/master.vue'),
		title: 'qcommerce.sidebar.adminProductEdit',
		icon: 'fas fa-boxes',
		authenticated: true,
	},
	//Categories list
	categories: {
		permission: 'icommerce.categories.index',
		activated: true,
		path: '/ecommerce/product-categories',
		name: 'qcommerce.admin.categories',
		page: () => import('@imagina/qcommerce/_pages/admin/category/index'),
		layout: () => import('@imagina/qsite/_layouts/master.vue'),
		title: 'qcommerce.sidebar.adminCategories',
		icon: 'fas fa-layer-group',
		authenticated: true,
	},
	//Product options
	options: {
		permission: 'icommerce.options.index',
		activated: true,
		path: '/ecommerce/product-options',
		name: 'qcommerce.admin.options',
		page: () => import('@imagina/qcommerce/_pages/admin/option/index'),
		layout: () => import('@imagina/qsite/_layouts/master.vue'),
		title: 'qcommerce.sidebar.adminOptions',
		icon: 'fas fa-cogs',
		authenticated: true,
	},
	//Product options values
	optionValues: {
		permission: 'icommerce.optionvalues.index',
		activated: true,
		path: '/ecommerce/product-options-values',
		name: 'qcommerce.admin.options.values',
		page: () => import('@imagina/qcommerce/_pages/admin/optionValues/index'),
		layout: () => import('@imagina/qsite/_layouts/master.vue'),
		title: 'qcommerce.sidebar.adminValues',
		icon: 'fas fa-stream',
		authenticated: true,
	},
	// Payment Methods
	paymentMethods: {
		permission: 'icommerce.payment-methods.manage',
		activated: true,
		path: '/payment-methods',
		name: 'qcommerce.admin.payment.methods',
		page: () => import('@imagina/qcommerce/_pages/admin/paymentMethods/index'),
		layout: () => import('@imagina/qsite/_layouts/master.vue'),
		title: 'qcommerce.sidebar.adminPaymentMethods',
		icon: 'fas fa-money-bill-wave',
		authenticated: true,
	},
	// Shipping Methods
	shippingMethods: {
		permission: 'icommerce.shipping-methods.manage',
		activated: true,
		path: '/shipping-methods',
		name: 'qcommerce.admin.shipping.methods',
		page: () => import('@imagina/qcommerce/_pages/admin/shippingMethods/index'),
		layout: () => import('@imagina/qsite/_layouts/master.vue'),
		title: 'qcommerce.sidebar.adminShippingMethods',
		icon: 'fas fa-shipping-fast',
		authenticated: true,
	},
  orders: {
		permission: 'icommerce.orders.index',
    activated: true,
    path: '/orders',
    name: 'qcommerce.admin.shipping.orders.index',
    page: () => import('@imagina/qcommerce/_pages/admin/order/index'),
    layout: () => import('@imagina/qsite/_layouts/master.vue'),
    title: 'qcommerce.sidebar.adminOrders',
    icon: 'fas fa-box-open',
    authenticated: true,
  },
  order: {
    permission: 'icommerce.orders.index',
    activated: true,
    path: '/order/:id',
    name: 'qcommerce.admin.shipping.orders.show',
    page: () => import('@imagina/qcommerce/_pages/admin/order/show'),
    layout: () => import('@imagina/qsite/_layouts/master.vue'),
    title: 'qcommerce.sidebar.adminOrders',
    icon: 'fas fa-box-open',
    authenticated: true,
  },
  coupons: {
    permission: 'icommerce.coupons.index',
    activated: true,
    path: '/coupons',
    name: 'qcommerce.admin.coupons.index',
    page: () => import('@imagina/qcommerce/_pages/admin/coupons/index'),
    layout: () => import('@imagina/qsite/_layouts/master.vue'),
    title: 'qcommerce.sidebar.adminCoupons',
    icon: 'fas fa-ticket-alt',
    authenticated: true,
  },
  couponsCreate: {
    permission: 'icommerce.coupons.create',
    activated: true,
    path: '/coupons/create',
    name: 'qcommerce.admin.coupons.create',
    page: () => import('@imagina/qcommerce/_pages/admin/coupons/form'),
    layout: () => import('@imagina/qsite/_layouts/master.vue'),
    title: 'qcommerce.sidebar.createCoupons',
    icon: 'fas fa-ticket-alt',
    authenticated: true,
  },
  couponsEdit: {
    permission: 'icommerce.coupons.edit',
    activated: true,
    path: '/coupons/:id',
    name: 'qcommerce.admin.coupons.edit',
    page: () => import('@imagina/qcommerce/_pages/admin/coupons/form'),
    layout: () => import('@imagina/qsite/_layouts/master.vue'),
    title: 'qcommerce.sidebar.editCoupons',
    icon: 'fas fa-ticket-alt',
    authenticated: true,
  },
}
